frontend/hub/api.tsx

```diff
@@ -22,8 +22,12 @@ export function pulpAPI(strings: TemplateStringsArray, ...values: string[]) {
   return process.env.HUB_API_BASE_PATH + '/pulp/api/v3' + apiTag(strings, ...values);
 }
 
+export function getRepoURL() {
+  return 'https://localhost:3002' + process.env.HUB_API_BASE_PATH;
```

Comment:
```suggestion
  return window.location.origin + process.env.HUB_API_BASE_PATH;
```

so this can also work in production